### ISSUE TITLE
E-mail addresses with underscores are misrendered in Markdown

### ISSUE THREAD

**Issue by charmander** [Weasyl/weasyl#277]
```markdown
_example_@example.com
```
renders as
```html
<em>exa<a href="…">_example_@example.com</a>@example.com</em>
```

This should be fixed by upgrading to misaka 2.1.0+weasyl.1.

**Comment by charmander** [comment]
This can’t be fixed by upgrading to misaka 2.x, because it (Hoedown) removes an important feature we use from 1.x: lax spacing.

**Comment by kfkitsune** [comment]
Out of curiosity, is there a way to ascertain how much lax-spaced content is in the DB? And if so, if it can be fixed programmatically?

**Comment by charmander** [comment]
@kfkitsune Yep, we can compare renderings for all Markdown content in the database (which would also catch other differences). It could annoy people to not have lax spacing, though. (For example, when we didn’t require code blocks to be fenced, there were lots of accidental indented code blocks.)

I’d rather move to a visual editor right away than do a misaka 1 → 2 migration in between.
